app/models/manageiq/providers/redhat/infra_manager/host.rb

```diff
@@ -17,4 +17,8 @@ def verify_credentials(auth_type = nil, options = {})
 
     true
   end
+
+  def supports_quick_stats?
```

Comment:
@durandom hmm does this work? I mean when the block in "supports" returns false does it cause it to return false? I thought you had to add "unsupported_reason"